src/aibtc-dao/extensions/bank-account/public/deposit-stx.ts

```diff
@@ -84,6 +94,8 @@ async function main() {
     network: networkObj,
     nonce: nextPossibleNonce,
     senderKey: key,
+    postConditionMode: PostConditionMode.Allow,
```

Comment:
We should use `PostConditionMode.Deny` so that the PC we set are evaluated against the TX events.